settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'medic-android'
\ No newline at end of file
+rootProject.name = 'medic-android'
```

Comment:
cht-android ?

Reply:
Yea, I don't know if we want to change those too. If we want the APK names to change and everything.

Reply:
I looked into this as well, but in the end, wanted to play it safe.

Reply:
Oh I see you added the question [here](https://github.com/medic/cht-core/issues/6586#issuecomment-848877731) whether to rename the APKs or not. I would say yes but lets wait Gareth's answer.